web/src/components/pages/contribution/speak/speak.tsx

```diff
@@ -519,8 +520,8 @@ class SpeakPage extends React.Component<Props, State> {
                     outline
                     rounded
                     className={getTrackClass('fs', 'exit-submit-clips')}
-                    onClick={() => {
-                      if (this.upload()) onConfirm();
```

Comment:
I'm confused by this change; we still need to call `onConfirm` eventually. Please re-review my previous feedback and let me know if you have questions.

Reply:
This will automatically close the prompt when the clip is uploaded and the modal is accepted that is what is required and to be done by the function. By this, we don't have to call onConfirm to close the prompt.
If this is not a satisfactory change, I will try with your previous feedback.

Reply:
![Screencast 2020-03-26 21_27_48](https://user-images.githubusercontent.com/43513114/77668574-95b45100-6fa9-11ea-98d1-f7053dd49fea.gif)